htdocs/class/xoopseditor/tinymce/tinymce/jscripts/tiny_mce/plugins/xoopsimagemanager/xoopsimagebrowser.php

```diff
@@ -71,19 +73,19 @@
 }
 
 if (isset($_GET['op'])) {
-    $op = trim($_GET['op']);
+    $op = Request::getString('op', '', 'GET');
```

Comment:
is trim used
1) only for cast a string?
2) or to remove really leading/trailing blanks?
if 2) then IMO it should be
$op = trim(Request::getString('op', '', 'GET'));

Reply:
trim() is not needed, because Request::getString() calls getVar() which then calls "cleanVar()" which includes trim():
```php
        // If the no trim flag is not set, trim the variable
        if (!($mask & static::MASK_NO_TRIM) && is_string($var)) {
            $var = trim($var);
        }
```

Reply:
there are some` trim(Request::getString(...` in the other files, I have made a comment